src/components/sidebar/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { injectIntl } from 'react-intl';
+import { object } from 'prop-types';
+
+import menuIcon from 'assets/images/menu-icon.svg';
+import closeIcon from 'assets/images/close.svg';
+import Menu from './Menu';
+
+import './styles.scss';
+
+const SideBar = ({ intl }) => {
+  const [visible, setVisible] = useState(false);
+  return visible ? (
+    <div className="menu">
+      <button className="simple-btn" type="button" onClick={() => setVisible(!visible)}>
+        <img src={closeIcon} alt="menu" />
+      </button>
+      <Menu intl={intl} />
+    </div>
+  ) : (
+    <div className="menu">
+      <button className="simple-btn" type="button" onClick={() => setVisible(!visible)}>
+        <img src={menuIcon} alt="menu" />
+      </button>
+    </div>
```

Comment:
You can move the condition to inside and do the following:

```suggestion
  return (
    <div className="menu">
      <button className="simple-btn" type="button" onClick={() => setVisible(!visible)}>
        <img src={visible? closeIcon: menuIcon} alt="menu" />
      </button>
      {visible && <Menu intl={intl} />}
    </div>
  )
```

Because the only things that change are the _src_ on the image and render or not the `Menu`, the rest is the same 🙂

Reply:
changed thanks!